ibis/spark/tests/test_client.py

```diff
@@ -0,0 +1,28 @@
+import ibis
+
+
+def test_list_databases(client_empty):
+    dbs = client_empty.list_databases()
+    assert dbs == ['default']
+
+
+def test_list_tables(client_simple):
+    tables = client_simple.list_tables()
+    assert tables == ['t']
+
+
+def test_get_schema(client_simple):
+    schema = client_simple.get_schema('t')
+    assert schema == ibis.schema([('foo', 'int64'),
+                                  ('bar', 'string')])
+
+# TODO
+# def test_table(client_simple):
```

Comment:
I would delete this for now or mark it as `xfail` so that you don't forget about it.